message_definitions/v1.0/ras_a.xml

```diff
@@ -4,6 +4,30 @@
   <version>0</version>
   <dialect>0</dialect>
   <enums>
+    <enum name="MAV_AUTOPILOT">
```

Comment:
I think instead of this enumeration we should have a message that supports sending a CAGE code

Reply:
let's consider that for v1.2.